lib/spack/docs/known_issues.rst

```diff
@@ -35,3 +35,36 @@ A workaround is to explicitly activate the variant related to the dependency:
    $ spack install hdf5+mpi ^openmpi
 
 See https://github.com/LLNL/spack/issues/397 for further details.
+
+
+---------------------------------
+``spack extensions`` doesn't work
+---------------------------------
+
+**Status:** Up for grabs if you want to try to fix it
+
+Spack provides an ``extensions`` command that lists all available extensions
+of a package, the ones that are installed, and the ones that are already
+activated. This is very useful in conjunction with ``spack activate``.
+Unfortunately, this command no longer works:
+
+.. code-block:: console
+
+   $ spack extensions python
+   ==> python@2.7.13%clang@8.0.0-apple~tk~ucs4 arch=darwin-sierra-x86_64 -ckrr4mg has no extensions.
+
+
+See https://github.com/LLNL/spack/issues/2895 for further details.
+
+
+----------------------------
+``spack setup`` doesn't work
+----------------------------
```

Comment:
I would also like to see #2664 merged. However, I've never used the command and don't know enough about the files that it touches to merge it myself. Until @tgamblin merges #2664, this documentation will be useful for preventing more issues like #3795 from popping up.